lib/hacluster.pm

```diff
@@ -11,14 +11,14 @@ use base Exporter;
 use Exporter;
 use strict;
 use warnings;
-use version_utils 'is_sle';
-use Scalar::Util 'looks_like_number';
+use version_utils qw(is_sle);
```

Comment:
Is it needed? Is it not `qw` more useful if we need to import more than one method?

Reply:
I generally prefer `qw()` even when it's only one for consistency. Let's say we have a line like:
```
use module 'fun1';
```
But we then modify it to add another:
```
use module qw(fun1 fun2);
```
So it's not only adding to it, but changing the way the included functions are imported. If it had been `qw(fun1)` from the start, the change in the line would've been simply 4 characters.

There was even an attempt in the past to make `qw()` mandatory even when including only one function. Not sure why it was dropped.